.github/actions/setup-node/action.yaml

```diff
@@ -0,0 +1,29 @@
+name: "CI setup"
+inputs:
+  node-version:
+    description: "Node version"
+    default: 16.15
+  pnpm-version:
+    description: "PNPM version"
+    default: latest
```

Comment:
hmm not sure.  The latest pnpm version should be stable.  Swayswap pins the version to 7, but fuels-wallet and fuel-ui use latest.  Thoughts @luizstacio @pedronauck

Reply:
If the project is not suitable to be constantly updated, it's worth pinning the version to avoid it breaking. Ex.: In the Wallet, we are continually updating the project and pnpm version locally also, in this way makes sense to use the latest.